app/controllers/styleguide_controller.rb

```diff
@@ -32,4 +32,8 @@ def pages_locator_invalid
   def pages_locator_location
     render template: 'styleguide/pages/locator_location'
   end
+
+  def pages_locator_location_call_centre
```

Comment:
'Locator' isn't a domain term that exists anywhere else in this application. We call them "Locations", you can **search** for a location and you can **view** a location.